src/kirby/components/list/list.component.tns.scss

```diff
@@ -51,3 +51,15 @@ RadListView {
     }
   }
 }
+
```

Comment:
The styling on native doesn't seem to be that beauty, this should be looked into before merging

Reply:
completely agree